change/@ni-nimble-components-6bd07a0a-7f59-4440-989a-516fca7124bf.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Create WaferMap component",
```

Let's be more precise in case someone sees the changelog and gets excited to start using it before it's ready.
```suggestion
  "comment": "Create placeholder WaferMap component",
```